lear-db/test_data/requirements/bcregistry-libraries.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/bcgov/business-schemas.git@2.5.11#egg=registry_schemas
```

Comment:
Should this move to the current version??

Reply:
Oh nice catch - I will update this to always get the current one